EventsExpress.Db/Migrations/20220307113046_UserMoreInfoUpdated.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EventsExpress.Db.Migrations
```

Comment:
Please, combine the two migrations into a single one. There is no point in having multiple migrations in one commit.

Reply:
Migrations `20211115110227_AddUserMoreInfo` and `20220307113046_UserMoreInfoUpdated`?